server/openapi_environment.go

```diff
@@ -173,10 +173,6 @@ func (s *Server) convertToOpenAPIEnvironment(ctx context.Context, env *api.Envir
 	if err != nil {
```

Comment:
We can entirely remove the old environment OpenAPI now (I can do this in another PR

Reply:
Done. Removed.